content/en/docs/tasks/network/extend-service-ip-ranges.md

```diff
@@ -291,11 +289,14 @@ spec:
   matchConstraints:
     resourceRules:
     - apiGroups:   ["networking.k8s.io"]
-      apiVersions: ["v1","v1beta1"]
+      apiVersions: ["v1"]
       operations:  ["CREATE", "UPDATE"]
       resources:   ["servicecidrs"]
   validations:
   - expression: "object.metadata.name == 'kubernetes'"
+    message: "only allow changes on the default servicecidr"
+  - expression: "request.userInfo.username == 'system:apiserver' && 'system:masters' in request.userInfo.groups"
+    message: "only apiserver can update and create servicecidrs"
```

Comment:
These messages don't read correctly in the context of the error the user will get. It should be more "you can't X".

Also, is the groups check really needed? Seems like the `system:apiserver` check should be sufficient...

Reply:
updated the messages